test/runtests.jl

```diff
@@ -23,6 +23,13 @@ end
 
 struct S end
 
+module A
+  using DataFrames, BSON
+  d = DataFrame(a = 1:10, b = 5:14)
+  # this is how the test file was generated
+  #bson("test_25_dataframe.bson", Dict(:d=>d))
```

Comment:
Should this be uncommented? Let ci generate it and load on the different OSs?

Reply:
Sure I can do that too. The trend I saw in the existing tests was to have the file already written.